### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by Kya-Edmeade**

### CODE REVIEW

**Review by sourcesmith — CHANGES_REQUESTED** [review]
Remove all the .DS_store files  - we do not need Mac scribblings all over the code base.  This should be in the .gitignore really.

**Review by sourcesmith — CHANGES_REQUESTED** [review]
Other than the specific comments.  General ones are:
No logging.
Little assertion of arguments.
Possible excessive catching of exceptions.

File: src/main/java/com/globallogic/amcr/repository/impl/layoutcomponent/LayoutDaoImpl.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        return layoutMapper.get(id);
+    }
+
+    public Layout getPage(String name) {
+        return layoutMapper.getPage(name);
+    }
+
+
+    public List<Layout> getAll() {
+        return layoutMapper.getAll();
```
[the remark this answers is not in the document]
Comment (Kya-Edmeade) [reply src/main/java/com/globallogic/amcr/repository/impl/layoutcomponent/LayoutDaoImpl.java:45]: I'm not sure what you mean by this comment

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -15,8 +15,13 @@ spring.mail.port=${AMCR_MAIL_PORT}
...
 spring.servlet.multipart.file-size-threshold=2KB
-spring.servlet.multipart.max-file-size=200MB
-spring.servlet.multipart.max-request-size=215MB
+spring.servlet.multipart.max-file-size=50MB
+spring.servlet.multipart.max-request-size=65MB
+
+# S.A - Whether to resolve the multipart lazily at the time of file or parameter access.
+spring.servlet.multipart.resolve-lazily=true
+# S.A - All files uploaded through the API will be stored here.
+file.upload-dir=/Users/siso/Desktop/Uploads
```
[the remark this answers is not in the document]
Comment (Kya-Edmeade) [reply src/main/resources/application.properties:24]: I believe this is relating to Stanislavs @S1sO0 part

File: src/main/java/com/globallogic/amcr/controller/layoutcomponent/LayoutController.java [hunk 3]
```diff
@@ -15,37 +18,52 @@
...
 
-    @GetMapping("/")
-    public List<Layout> getALl() {
-        return layoutService.getAll();
+    @GetMapping(value = "/", produces = "application/json")
+    public List<Layout> getAll() {
+        LOG.debug("Controller getting all layouts");
+        try {
+            return layoutService.getAll();
+        } catch (Exception e) {
```
Comment (sourcesmith) [inline src/main/java/com/globallogic/amcr/controller/layoutcomponent/LayoutController.java]: You should not really need this handler.

**Pull request merged**